CHANGELOG.md

```diff
@@ -4,6 +4,144 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project will adhere to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) starting v1.0.0.
 
+## 1.1.1 - [Unreleased][Unreleased-v1.1.1]
+[Unreleased-v1.1.1]: https://github.com/dgraph-io/dgraph/compare/v1.1.0...master
+
+### Changed
+
+- **Breaking changes for expand() queries**
+  - Remove `expand(_forward_)` and `expand(_reverse_)`. ([#4119][])
+  - Change `expand(_all_)` functionality to only include the predicates in the type. ([#4171][])
+- Add support for Go Modules. ([#4146][])
+- Simplify type definitions: type definitions no longer require the type (string, int, etc.) per field name. ([#4017][])
+- Adding log lines to help troubleshoot snapshot and rollup. ([#3889][])
+- Add `--http` flag to configure pprof endpoint for live loader. ([#3846][])
+- Use snappy compression for internal gRPC communication. ([#3368][])
+- Periodically run GC in all dgraph commands. ([#4032][], [#4075][])
+- Exit early if data files given to bulk loader are empty. ([#4253][])
+- Add support for first and offset directive in has function. ([#3970][])
+- Pad encData to 17 bytes before decoding. ([#4066][])
+- Remove usage of deprecated methods. ([#4076][])
+- Show line and column numbers for errors in HTTP API responses. ([#4012][])
+- Do not store non-pointer values in sync.Pool. ([#4089][])
+- Verify that all the fields in a type exist in the schema. ([#4114][])
+
+Enterprise features:
+
+- ACL: Disallow schema queries when an user has not logged in. ([#4107][])
+
+### Added
+
+- Support `@cascade` directive at subqueries. ([#4006][])
+- Support `@normalize` directive for subqueries. ([#4042][])
+- Support `val()` function inside upsert mutations (both RDF and JSON). ([#3877][], [#3947][])
+- Return mutated UIDs which are part of an upsert operation in the mutation response. ([#4145][])
```

Comment:
We should not release this change. I am remvoing this as part of my PRs.

Reply:
The line has been removed.